src/ONYXKEYS.js

```diff
@@ -37,12 +37,18 @@ export default {
     // an international code
     COUNTRY_CODE: 'countryCode',
 
+    // Contains all the users settings for the Settings page and sub pages
+    USER: 'user',
+
     // Information about the current session (authToken, accountID, email, loading, error)
     SESSION: 'session',
     IS_SIDEBAR_SHOWN: 'isSidebarShown',
     IS_SIDEBAR_ANIMATING: 'isSidebarAnimating',
     BETAS: 'betas',
 
+    // NVP keys
+    NVP_PAYPALMEADDRESS: 'nvp_paypalMeAddress',
```

Comment:
Let me know if you'd rather this be `NVP_PAYPAL_ME_ADDRESS` - just went with consistency for now.

Reply:
It does look better to me.